doc/en/user/source/community/opensearch-eo/STAC.rst

```diff
@@ -146,3 +146,120 @@ Support for the `STAC Datacube Extension <https://github.com/stac-extensions/dat
 				</li> 
 			</ul>
 	</li>
+
+
+Layer information support
+-------------------------
+
+The OpenSearch REST admin API allows to create and manage layers, and the STAC API exposes them as collections.
+
+Layers are kept in the ``layers`` attribute of a collection, each one exposes the following information:
+
+* All ``collection_layers`` attributes (``workspace``, ``layer``, ``bands``, and so on)
+* The layer ``title`` and ``description`` from the GeoServer own configuration
+* A list of ``styles``, matching the styles associated to the layer. The first one is the default style. Each style contains the follwoing properties:
+  - ``name``: the style name
+  - ``title``: the style title
+* A ``services`` object with attributes matching a lowercase service name, e.g., ``wms``, ``wcs``, ``wmts``, each one being an object with the following two fields:
+    - ``enabled``: true or false (might depend on the layer configuration too)
+    - ``formats``: the formats supported, for that layer, by the main service output (e.g., maps, coverages, tiles)
+
```

Comment:
It would be helpful for context to link back to the applicable STAC extension: https://github.com/stac-extensions/web-map-links